lib/logic.rb

```diff
@@ -0,0 +1,149 @@
+class Board
+  attr_accessor :board
+
+  def initialize
+    @turn = true
+    @board = [
```

Comment:
I think this is the same array you are using in your display right? I suggest you separate any code/logic that has to do with display from your classes. Generally we would use an array for @board attribute here to equal something like [1,2....9].